unliftio/src/UnliftIO/Async.hs

```diff
@@ -536,12 +569,16 @@ instance (Monoid a, MonadUnliftIO m) => Monoid (Conc m a) where
 
 -- | Flattened structure, either Applicative or Alternative
 data Flat a
-  = FlatApp (FlatApp a)
+  = FlatApp !(FlatApp a)
```

Comment:
Was the strictness here for correctness or performance? Just curious.

Reply:
We were looking it from a performance angle